code/src/components/styled/home.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components'
+
+export const AppContainer = styled.div`  
+  display:flex;
+  flex-direction: column;
+  align-items:flex-start;
+  justify-content:flex-start;
+  height: 667px;
```

Comment:
Your app is not ready for very busy people like me :D
![Image](https://user-images.githubusercontent.com/50926159/116782103-914d4800-aa87-11eb-9d24-c1913fe928f1.png)